HOMER_LP-obs-studio-cleanup-prepare-1.5.0_HES.md

```
@@ -0,0 +1,216 @@
+# If PASS, upload
+gsutil cp /tmp/dryrun.prod.json gs://ropi-aoss-backups/dryruns/dryrun-prod-$(date -u +%Y%m%dT%H%M%SZ).json
+```
+
+### 2. Post Approver Notification
+Copy the PR comment template from `HES/LP-obs-studio-cleanup-apply-1.5.0-APPROVER-READY.md` Section 7 and post to the Apply PR.
+
+### 3. Collect Approvals
+Ensure all 4 approvers sign off in the PR comment format.
+
+### 4. When Ready, Paste Values for Apply LP Relay
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)